torch/fx/experimental/normalize.py

```diff
@@ -117,3 +120,52 @@ def _args_kwargs_to_normalized_kwargs(self, sig : inspect.Signature, args : Tupl
             new_kwargs[param] = bound_args.arguments[param]
 
         return new_kwargs
+
+
+class NormalizeOperators(AnnotateTypesWithSchema):
+    """
+    Normalize callsites that are different ways of "spelling" the same
+    invocation into a single, canonical call. Currently supports:
+
+    1. Normalize operators (e.g. operator.add) to the `torch` ops they
+       ultimately invoke (e.g. torch.add) when it is possible to statically
+       reason that
+
+    Example usage:
+
+        m = torchvision.models.resnet18()
+
+        traced = torch.fx.symbolic_trace(m)
+
+        traced = NormalizeOperators(traced).transform()
+    """
+    binary_magic_method_remap : Dict[Callable[[Any, Any], Any], Callable[[Any, Any], Any]] = {
+        torch.add : operator.add,
+        torch.mul : operator.mul,
+        torch.sub : operator.sub,
+        torch.div : operator.truediv,
+        torch.floor_divide : operator.floordiv,
+        torch.remainder : operator.mod,
```

Comment:
double check % does the same thing, since they are not exactly the same operator (requires testing with negative numbers) (https://rob.conery.io/2018/08/21/mod-and-remainder-are-not-the-same/)

Reply:
Would the argument for the `operator.mod -> torch.remainder` direction still apply here? This is going to get dispatched down to `torch.remainder` judging by what `python_variable_methods` does so it it consistent with PT?